client/extensions/wp-super-cache/components/debug/index.jsx

```diff
@@ -0,0 +1,201 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { pick } from 'lodash';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormTextInput from 'components/forms/form-text-input';
+import FormToggle from 'components/forms/form-toggle/compact';
+import SectionHeader from 'components/section-header';
+import WrapSettingsForm from '../wrap-settings-form';
+
+class DebugTab extends Component {
+	static propTypes = {
+		fields: PropTypes.object,
+		translate: PropTypes.func.isRequired,
+	};
+
+	static defaultProps = {
+		fields: {},
+	};
+
+	render() {
+		const {
+			fields: {
+				wp_super_cache_debug,
+				wp_cache_debug_ip,
+				wp_super_cache_comments,
+				wp_super_cache_front_page_check,
+				wp_super_cache_front_page_clear,
+				wp_super_cache_front_page_text,
+				wp_super_cache_front_page_notification,
+			},
+			handleAutosavingToggle,
+			handleChange,
+			handleSubmitForm,
+			isRequesting,
+			isSaving,
+			translate,
+		} = this.props;
+
+		return (
+			<div>
+				<SectionHeader label={ translate( 'Debug' ) }>
+					<Button
```

Comment:
I recently discovered the [`<FormButton>` component](https://github.com/Automattic/wp-calypso/blob/master/client/components/forms/form-button/index.jsx) which would be a better fit here instead of `<Button>`. It takes care of setting the button text based on the value of the `isSubmitting` prop.